src/aws.js

```diff
@@ -20,8 +20,8 @@ function buildUserDataScript(githubRegistrationToken, label) {
       '#!/bin/bash',
       'mkdir actions-runner && cd actions-runner',
       'case $(uname -m) in aarch64) ARCH="arm64" ;; amd64|x86_64) ARCH="x64" ;; esac && export RUNNER_ARCH=${ARCH}',
-      'curl -O -L https://github.com/actions/runner/releases/download/v2.280.3/actions-runner-linux-${RUNNER_ARCH}-2.280.3.tar.gz',
-      'tar xzf ./actions-runner-linux-${RUNNER_ARCH}-2.280.3.tar.gz',
+      'curl -O -L https://github.com/actions/runner/releases/download/v2.284.0/actions-runner-linux-${RUNNER_ARCH}-2.284.0.tar.gz',
+      'tar xzf ./actions-runner-linux-${RUNNER_ARCH}-2.284.0.tar.gz',
```

Comment:
the version update here is not required, could be pulled out - would still be nice for this to just be automatically the most recent release (https://github.com/machulav/ec2-github-runner/issues/16)